modules/ip4s/build.sbt

```diff
@@ -0,0 +1,20 @@
+import Dependencies.Version._
+
+name := "pureconfig-ip4s"
+
+crossScalaVersions := Seq(scala212, scala213)
+
+libraryDependencies ++= Seq("com.comcast" %% "ip4s-core" % "3.0.4")
+
+developers := List(
+  Developer("geirolz", "David Geirola", "david.geirolz@gmail.com", url("https://github.com/geirolz"))
+)
+
+osgiSettings
```

Comment:
Can you please remove these keys? This won't merge cleanly with master as we dropped support for OSGi in #1202.